.travis.yml

```diff
@@ -4,4 +4,5 @@ node_js:
 
 script:
   - npm run lint
-  - npm run test
+  - npm run test-root
```

Comment:
Just rename this to test and don't worry about test-functions yet.

This setup isn't quite right as you are still using the node_modules installed at root to run the tests for things in /functions.

But let's get what we have in first.

Reply:
Gotcha, since the next thing Nick and I are working on are unit tests for the cloud functions, we will look more into that